app/src/test/java/de/grobox/transportr/utils/DateUtilsTest.kt

```diff
@@ -0,0 +1,191 @@
+/*
+ *    Transportr
+ *
+ *    Copyright (c) 2013 - 2018 Torsten Grote
+ *
+ *    This program is Free Software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as
+ *    published by the Free Software Foundation, either version 3 of the
+ *    License, or (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.grobox.transportr.utils
+
+import android.content.Context
+import android.content.res.Resources
+import android.view.View
+import de.grobox.transportr.R
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.*
+import org.mockito.MockitoAnnotations
+import java.util.*
+
+
+class DateUtilsTest {
+
+    @Mock
+    lateinit var context: Context
+    @Mock
+    lateinit var resources: Resources
+
+    private fun minuteToMillis(minutes: Float): Long {
+        return (minutes * android.text.format.DateUtils.MINUTE_IN_MILLIS).toLong()
+    }
+
+    @Test
+    fun millisToMinutes() {
+        Assert.assertEquals(0, DateUtils.millisToMinutes(minuteToMillis(0f)))
```

Comment:
You could import the methods directly, so removing `Assert.` and `DateUtils.`. Makes the code a bit smaller and easier to read, I'd say.

Reply:
Sounds like a good idea :+1: